### ISSUE TITLE
Allow custom property-sort-order file

### ISSUE THREAD

**Issue by ncoden** [sasstools/sass-lint#602]
It could be great to allow a custom property sort file like:
```
.sass-lint
└── property-sort-order.yml
.sass-lint.yml
```
With:
```yaml
# .sass-lint.yml
rules:
  property-sort-order:
    - 1
    -
      order: .sass-lint/property-sort-order.yml
```
Or
```yaml
...
      order: property-sort-order
```

**Comment by BPScott** [comment]
We can already define custom sort orders within .sass-lint.yml [using the order property](https://github.com/sasstools/sass-lint/blob/develop/docs/rules/property-sort-order.md#custom-sort-orders)

```yaml
property-sort-order:
  - 1
  - order:
      - border
      - display
      - color
```

Is this good enough? As I'm not sure what value is gained by including that array of items in an external file.

**Comment by DanPurdy** [comment]
I don't think this is something we would pursue it opens up too many edge cases and potential importing issues, as @BPScott says you can already define this order and you can just share this config amongst all your projects. Once we get to including configs via packages like you can with eslint airbnb standards etc then I think this would become a moot point?

**Comment by ncoden** [comment]
@DanPurdy @BPScott Anything new about this issue ? I still find this very useful.
